backend-api/src/functions/asyncCredential/asyncCredentialHandler.ts

```diff
@@ -221,9 +207,9 @@ export async function lambdaHandler(
 
   const writeEventResult = await eventService.writeGenericEvent({
     eventName: "DCMAW_ASYNC_CRI_START",
-    sub,
+    sub: requestBody.sub,
     sessionId,
-    govukSigninJourneyId: govuk_signin_journey_id,
+    govukSigninJourneyId: requestBody.govuk_signin_journey_id,
```

Comment:
Opted to use these values inline as opposed to deconstructing above to as there are only two values needed - avoids adding an extra line